### ISSUE TITLE
Inline math at the beginning of a sentence

### ISSUE THREAD

**Issue by kiryph** [dbmrq/vim-bucky#3]
Consider following text and its current result from vim-bucky with `tw=80`:
```tex
At vero eos et accusam et justo duo dolores et ea rebum.
Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit
  amet. $E$ is a quantity ipsum dolor sit amet, consectetur adipisicing elit,
  sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut
  aliquip ex ea commodo consequat.
```
However, `$E$` should be on a new line:
```tex
At vero eos et accusam et justo duo dolores et ea rebum.
Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit
  amet.
$E$ is a quantity ipsum dolor sit amet, consectetur adipisicing elit, sed do
  eiusmod tempor incididunt ut labore et dolore magna aliqua.
Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut
  aliquip ex ea commodo consequat.
```

Just for comparison latexindent version 3.5 handles this partially correctly:
```
❯ latexindent demo.tex -m -l=localSettings-lst269.yaml -w
At vero eos et accusam et justo duo dolores et ea rebum.
Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit
  amet.
$E$ is a quantity ipsum dolor sit amet, consectetur adipisicing elit,
sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut
  aliquip ex ea commodo consequat.
```
the subordinate clause `sed do eiusmod ...` should be indented.

**Comment by kiryph** [comment]
Thanks for the update:
```
~/.vim/pack/minpac/start/vim-bucky master
❯ g lg
* 127543b - (HEAD -> master, origin/master, origin/HEAD) Fix tiny problem with lines ending in \ (8 hours ago) [Daniel M]
* a86b579 - Indent equation environments Closes #4 (11 hours ago) [Daniel M]
* 461f73a - Support starting new sentences with math Closes #3 (13 hours ago) [Daniel M]
* 2c245c4 - Support tw=0 for markdown too (25 hours ago) [Daniel M]
* df9e31b - Support tw=0 Closes #2 (25 hours ago) [Daniel M]
* a735118 - Improve indentation after comments (28 hours ago) [Daniel M]
* c0af50f - (grafted) First commit (2 days ago) [Daniel M]
```

However, it does the same as latexindent: no indentation for `sed do eiusmod ...`.
![screen shot 2018-08-28 at 09 38 31](https://user-images.githubusercontent.com/371756/44707938-26f9d380-aaa6-11e8-866f-acb17ec90751.png)

**Comment by dbmrq** [comment]
Ok, I think I got it now. :)
